lib/round.rb

```diff
@@ -0,0 +1,50 @@
+class Round
+  attr_reader :deck,
+              :turns
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+  end
+
+  def current_card
+    @deck.cards[@turns.count]
+  end
+
+  def take_turn(answer)
+    new_turn = Turn.new(answer, @deck.cards[@turns.count])
+    @turns << new_turn
+    return new_turn
+  end
+
+  def number_correct
+    correct_count = 0
+    @turns.each do |turn|
+      if turn.feedback == "Correct!"
```

Comment:
This will definitely work, you could also look at your `correct?` method since that will automatically return true or false for you